notifications/hub.go

```diff
@@ -36,6 +36,11 @@ func NewHub() *Hub {
 	}
 }
 
+// SubscriberCount returns the number of subscribers
+func (h *Hub) SubscriberCount() int {
+	return len(h.subscribers)
```

Comment:
I thought I could get away without it but you're right, it's part of the public API. Have added it in.